### PR TITLE
Header

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Update design styles for header and refactor sidebar navigation to a list.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: apps/foundation-demo/src/app/core/sidebar/containers/sidebar/sidebar.component.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
   <span class="sidebar__close" (click)="onClose()"><i class="fa fa-times"></i></span>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">Women's</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">Men's</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">Accessories</a>
-  <a daff-sidebar-item (click)="onClose()" routerLink="/">Sale</a>
+  <ul class="sidebar__list">
+    <li><a daff-sidebar-item (click)="onClose()" routerLink="/">Women's</a></li>
+    <li><a daff-sidebar-item (click)="onClose()" routerLink="/">Men's</a></li>
+    <li><a daff-sidebar-item (click)="onClose()" routerLink="/">Accessories</a></li>
+    <li><a daff-sidebar-item (click)="onClose()" routerLink="/">Sale</a></li>
```
Comment (lderrickable) [inline apps/foundation-demo/src/app/core/sidebar/containers/sidebar/sidebar.component.html]: I think this should use the `daff-list` component instead, which is part of the footer pull request.

**Comment by lderrickable** [comment]
I think the rebase could be ugly here. I think you started with an old version of the header branch or something

**Comment by xelaint** [comment]
> I think the rebase could be ugly here. I think you started with an old version of the header branch or something

Should I just delete this branch and branch off of footer for my changes?

**Comment by lderrickable** [comment]
> > I think the rebase could be ugly here. I think you started with an old version of the header branch or something
> 
> Should I just delete this branch and branch off of footer for my changes?

Probably. Though I'd branch off of develop rather than footer

**Comment by damienwebdev** [comment]
@lderrickable this can be cherry-picked into a new branch off develop fairly easily there's only two commits.

**Pull request closed without merging**